.gitignore

```diff
@@ -15,8 +15,11 @@
 *.cov
 a.out
 lib/
+test-dev/gen_mixer_data
+test-dev/gen_module_data
 test-dev/libxmp-covertest
-test-dev/libxmp-test
+test-dev/libxmp-tests
```

Comment:
While you're at it, change `test/libxmp-tests`to `test/libxmp-test` an remove non-existent `test/test8.raw`

Go ahead and apply with that change.

Reply:
Will do. I was under the impression another pull request did address that, but I can't find it.